4_de_analysis/de_analysis.R

```diff
@@ -0,0 +1,62 @@
+# functions to perform RNA-seq differential expression analysis using DESeq2
+# mostly copied from and inspired by:
+# https://github.com/greenelab/generic-expression-patterns/blob/master/generic_expression_patterns_modules/DE_analysis.R
```

Comment:
I'm glad this was useful:) I remember it took a while to understand what the appropriate inputs and settings were so I'm glad if that saved you some time

Reply:
Yes, very useful! Your code was so well documented, it definitely saved me some time.